### PR TITLE
MB-6206: ␡ GetAWSConfig

### PR DESCRIPTION
**Pull request by mdawn**
## [Remove Vault cmds from send_post_move_survey_email](https://dp3.atlassian.net/browse/MB-6206)

- Removes our artisan vault call
- Deletes depreciated `experimental.send-post-move-survey.env` in favor of `exp.send-post-move-survey.env`.

Screen Shot 2021-02-02 at 6 27 39 PM

Screen Shot 2021-02-02 at 6 30 11 PM
triggered the email in the console at `25 0 * * ? *` utc time

Screen Shot 2021-02-02 at 6 29 20 PM
It gave me this dot! 🎉

### CODE REVIEW

**Review by carterjones — APPROVED** [review]

File: docs/adr/index.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ This log lists the architectural decisions for DP3 Infrastructure.
 - [ADR-0056](0056-use-asdf-to-manage-golang-versions-in-development.md) - Use ASDF To Manage Golang Versions In Development
 - [ADR-0057](0057-lodash.md) - Deprecate use of lodash over time
 - [ADR-0058](0058-replace-loki-with-happo.md) - Use Happo for visual regression testing
+- [ADR-0059](0059-use-snapshot-to-cleanup-loadtesting.md) - Use snapshot to cleanup load testing
```
Comment (gidjin) [inline docs/adr/index.md:66]: I wonder how this was missed, I thought there was a build step to make sure this didn't happen. 🤔

File: cmd/milmove-tasks/send_post_move_survey_email.go [hunk 2]
```diff
@@ -95,10 +96,8 @@ func sendPostMoveSurvey(cmd *cobra.Command, args []string) error {
 
 	var session *awssession.Session
 	if v.GetBool(cli.DbIamFlag) || (v.GetString(cli.EmailBackendFlag) == "ses") {
-		verbose := cli.LogLevelIsDebug(v)
-		c, errorConfig := cli.GetAWSConfig(v, verbose)
-		if errorConfig != nil {
-			logger.Fatal(errors.Wrap(errorConfig, "error creating aws config").Error())
+		c := &aws.Config{
+			Region: aws.String(v.GetString(cli.AWSRegionFlag)),
```
Comment (monfresh) [inline cmd/milmove-tasks/send_post_move_survey_email.go:100]: Do we still have the ability to run this with debug log level, or some kind of verbose level?

**Review by gidjin — APPROVED** [review]
LGTM, thanks for fixing the adr index.

File: docs/adr/index.md (hunk 1 above)
Comment (monfresh) [reply docs/adr/index.md:66]: It was broken until I fixed the Truss pre-commit hooks.

File: cmd/milmove-tasks/send_post_move_survey_email.go (hunk 2 above)
Comment (mdawn) [reply cmd/milmove-tasks/send_post_move_survey_email.go:100]: Aah. Great question. 

This seems related to what you & Barry were talking about in [this convo](https://ustcdp3.slack.com/archives/CP6PTUPQF/p1611754875095700). I think I see what you're saying. 

For [`aws.config` specifically](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html), there's not a lot of ambiguity - you just wouldn't be able to create a session, and you'd hit that [NewSession error](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/) on line 102.

**Pull request merged**